### DISCUSSION TITLE
Multiple Snapshots

### DISCUSSION THREAD

**Discussion by dkornahrens** [tsightler/ring-mqtt#721] [Support Q&A]
I looked through the documentation and perhaps I missed it.  Is there a method to save multiple snapshots?  Like if I wanted to save the last ten from a specific camera or something?  Thank you in advance.

**Comment by tsightler** [comment]
No, you must do this through your own automations like you would with any other camera.

**Reply by dmkjr** [reply]
Ahh, I see. I see the last snapshot created in /media. Where does that mount from?  Haven’t used much of the media section, pretty empty other than the snapshots.

**Reply by tsightler** [reply]
There are multiple ways to get snapshots to disk with Home Assistant, but ring-mqtt does not store any snapshots on disk at all so I'm not sure where those are coming from.  If you want to save snapshots, you can use the standard Home Assistant automations to take Home Assistant snapshots of the snapshot camera provided by ring-mqt and the automation can save them however and wherever you like.

I don't really know much about /media either, by default on Home Assistant OS it's just a virtual mount to the free space on the primary disk you installed on.
